### ISSUE TITLE
possible bug in RuxInput react component

### ISSUE THREAD

**Issue by 321github123** [RocketCommunicationsInc/astro#104]
Hi,
 
When I set “invalid” to “false” in the  RuxInput react, the help text appears as error text (red).
 
 
![image](https://user-images.githubusercontent.com/94007578/140974716-80ac4248-bbe7-4feb-9413-70d55e80338d.png)

 
I modified the sandbox code here
 
https://astro-components.netlify.app/?path=/story/astro-uxds-patterns-forms-react--page
 
 
 
 
Code below:
 ```javascript
import './styles.css'
import { React, useState } from 'react'
import {
    RuxInput,
    RuxSelect,
    RuxCheckbox,
    RuxCheckboxGroup,
    RuxRadio,
    RuxRadioGroup,
    RuxSlider,
    RuxButton,
    RuxOption,
} from '@astrouxds/react'
export default function App() {
    const [firstName, setFirstName] = useState('')
    const [lastName, setLastName] = useState('')
    const [email, setEmail] = useState('')
    const [countryRegion, setCountryRegion] = useState('United States')
    const [options, setOptions] = useState('')
    const [range, setRange] = useState(50)
    const [things, setThings] = useState([])

    const handleSubmit = (e) => {
        e.preventDefault()
        alert(`
     First Name: ${firstName} \n
     Last Name: ${lastName} \n
     Email: ${email} \n
     Country/Region: ${countryRegion} \n
     Options: ${options} \n
     Things: ${things} \n
     Range: ${range} \n
    `)
    }
    const handleThings = (e) => {
        let arr = things
        arr.push(e.target.value)
        let unique = [...new Set(arr)]
        setThings(unique)
    }
    return (
        <div className="container">
                <div>
                    <RuxInput
                        label="Last Name"
                        value={lastName}
                        help-text='help-text'
                        error-text=''
                        invalid={false}
                    />
                </div>
        </div>
    )
}
```

**Comment by micahjones13** [comment]
Hey there,

Just wanted to give an update that we've identified the issue that is causing the error-text styling to be applied to the help-text when help-text is not an empty string but error-text is. We have a solution and will be implementing it here shortly.

Thanks for bringing this up! I will comment here again when the fix is deployed.

**Comment by 321github123** [comment]
Thanks for the update!

**Comment by markacianfrani** [comment]
we've just published 6.0.2 that hotfixes this. let us know if youre still having issues, thanks!
